concent-builder-vm/readme.txt

```diff
@@ -0,0 +1,47 @@
+# HOW TO SETUP A VAGRANT MACHINE
+
+1. Install VirtualBox package. 
+Package download from https://www.virtualbox.org/wiki/Linux_Downloads . 
+su 
+... 
+dpkg -i <PACKAGE_NAME> 
+
+
+2. Install Vagrant. 
+Download from: https://www.vagrantup.com/downloads.html 
+su 
+... 
+dpkg -i <PACKAGE_NAME> 
+
+
+3. Install VirtualBox provider inside Vagrant. 
+COMMAND:
+
+vagrant box add \
+  precise64 https://files.hashicorp.com/precise64.box
+
+4. Install image your machines. 
+Choose from: https://app.vagrantup.com/boxes/search?_ga=2.138139312.1235557729.1509978713-689630316.1509978713
```

Comment:
What's `_ga=2.138139312.1235557729.1509978713-689630316.1509978713` for? Is it some kind of tracking ID (does not seem to affect search results)? Please make sure you never commit stuff like this into the repository.